src/model/index.py

```diff
@@ -0,0 +1,18 @@
+import os
+from output.exceptions import *
+from commands.keys import Keys
+
+
+class Index:
+    def __init__(self, name, table_map):
```

Comment:
Index should take TableMetadata not a table_map. 
We represent the Table by TableMetadata.